bgpd/bgp_zebra.c

```diff
@@ -1355,6 +1351,7 @@ void bgp_zebra_announce(struct bgp_dest *dest, const struct prefix *p,
 
 	for (; mpinfo; mpinfo = bgp_path_info_mpath_next(mpinfo)) {
 		uint32_t nh_weight;
+		bool is_evpn;
```

Comment:
I see this being declared and set within the `for` loop, but I don't see it being used anyplace in this context? I might have missed it, though ...

Reply:
It's used as an argument to `update_ipv4nh_for_route_install`, `update_ipv6nh_for_route_install` and in line 1471. This flag is moved here from outside, so maybe these lines are not shown in the diff...